### DISCUSSION TITLE
General Discussion related to brainstorming

### DISCUSSION THREAD

**Discussion by GuyEternal** [GuyEternal/HUIM_OOPs#1] [General]
### **This is a general discussion related to brainstorming the implementation and the actual implementation.**

**Comment by GuyEternal** [comment]
@ankitsankhyan  @ManavJethva 
Should we add attribute related to total number of doctors in the hospital class so that all the base classes have them, and then equate the sum of all the doctors of every department (if it exists) to the total number of doctors?

**Comment by GuyEternal** [comment]
Let's discuss the file structure of the code @ankitsankhyan @ManavJethva

**Comment by GuyEternal** [comment]
Nobody is trying to take part here man
Please respond

**Reply by ManavJethva** [reply]
Arey bhai ispey jab code karna start karengey tab discussion karengey na

**Reply by GuyEternal** [reply]
Abhi se start kar lete he na

**Reply by GuyEternal** [reply]
Code khol ke dekh meine commits kar bhi diye he thode

**Reply by ManavJethva** [reply]
ok mai devansh ka laptop pehley set kar deta hoon fir start kartey hai

**Comment by GuyEternal** [comment]
Ek kam karo sab claim karo ki konse konse files tum complete karoge
fir me issues banunga sab ke liye aur tum implement karo unko aur fir Pull request bhej na sath me baith ke review karenge PRs aur fir merge kar denge


ya fir

one by one files kar sakte he
ki sab milke baitho aur ek file implement karo.

Konsa kare?

**Comment by ankitsankhyan** [comment]
|Classes      | Assignees    |
| :---        |          ---: |
| Hospital.hpp, Contact.hpp, Location.hpp, User.hpp | @GuyEternal   |
| Department.hpp, Private.hpp , Timing.hpp | @ManavJethva   |
| Public.hpp, Primary-Health-Centre.hpp , Secondary-Health-Centre.hpp , Tertiary-Health-Centre.hpp|@ankitsankhyan  |

**Reply by GuyEternal** [reply]
I am creating issues and assigning them accordingly.
